pkg/go/transformer/testcases_test.go

```diff
@@ -259,7 +259,12 @@ func loadModuleTestCases() ([]moduleTestCase, error) { //nolint:cyclop
 				}
 			}
 
-			testCase.ExpectedErrors = errors
+			// If we have 0 errors after removing the validation errors then just skip the test
+			if len(errors) == 0 {
+				testCase.Skip = true
```

Comment:
What do you mean after removing the valid actions?

Are you testing for those elsewhere?

Usually errors = 0 means we still test it but expect no errors

Reply:
This is because in Go we don't have any validation functionality, the 2 new test fixtures are purely validation errors so they won't error in Go. The existing one has some of the transform errors and syntax errors which do exist in Go